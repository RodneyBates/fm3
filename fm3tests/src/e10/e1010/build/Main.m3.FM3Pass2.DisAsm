 ItkEOF()
 ItkLeftEnd()
 ItkModuleLt(L1(16_1),[1,0])
 ItkOpenScopeLt(L1(16_1))
 ItkOpenDeclListLt(L8(16_8),[5,2])
 ItkTypeDeclLt([5,7])
 ItkDeclNo(D3,[5,7])
 ItkTypeDeclType([5,9])
 ItkTypeDeclRt([5,7])
 ItkOpenDeclListSep(L1(16_1),[7,0])
 ItkTypeDeclLt([7,7])
 ItkDeclNo(D4,[7,7])
 ItkTypeDeclType([7,9])
 ItkTypeDeclRt([7,7])
 ItkOpenDeclListSep(L2(16_2),[9,0])
 ItkTypeDeclLt([9,7])
 ItkDeclNo(D5,[9,7])
 ItkTypeDeclType([9,9])
 ItkTypeDeclRt([9,7])
 ItkOpenDeclListSep(L3(16_3),[11,0])
 ItkTypeDeclLt([11,7])
 ItkDeclNo(D7,[11,7])
 ItkTypeDeclType([11,9])
 ItkTypeDeclRt([11,7])
 ItkOpenDeclListSep(L4(16_4),[13,0])
 ItkTypeDeclLt([13,7])
 ItkDeclNo(D9,[13,7])
 ItkTypeDeclType([13,11])
 ItkFieldDeclListLt(L2(16_2),[14,6])
 ItkFieldDeclLt([14,6])
 ItkFieldDeclIdListLt(L1(16_1),[14,6])
 ItkDeclNo(D1,[14,6])
 ItkFieldDeclIdListRt(L1(16_1),[14,6])
 ItkFieldDeclType([14,9])
 ItkFieldDeclValue([15,4])
 ItkDeclValAbsent([15,4])
 ItkFieldDeclRt([14,6])
 ItkFieldDeclListSep(L1(16_1),[15,4])
 ItkFieldDeclLt([15,6])
 ItkFieldDeclIdListLt(L1(16_1),[15,6])
 ItkDeclNo(D2,[15,6])
 ItkFieldDeclIdListRt(L1(16_1),[15,6])
 ItkFieldDeclType([15,9])
 ItkFieldDeclValue([16,4])
 ItkDeclValAbsent([16,4])
 ItkFieldDeclRt([15,6])
 ItkFieldDeclListRt(L2(16_2),[14,6])
 ItkTypeDeclRt([13,7])
 ItkOpenDeclListSep(L5(16_5),[18,0])
 ItkTypeDeclLt([18,7])
 ItkDeclNo(D8,[18,7])
 ItkTypeDeclType([18,9])
 ItkTypeDeclRt([18,7])
 ItkOpenDeclListSep(L6(16_6),[20,0])
 ItkTypeDeclLt([20,7])
 ItkDeclNo(D6,[20,7])
 ItkTypeDeclType([20,9])
 ItkTypeDeclRt([20,7])
 ItkOpenDeclListSep(L7(16_7),[22,0])
 ItkTypeDeclLt([22,7])
 ItkDeclNo(D10,[22,7])
 ItkTypeDeclType([22,9])
 ItkTypeDeclRt([22,7])
 ItkOpenDeclListRt(L8(16_8),[5,2])
 ItkModuleBegin(L1(16_1),[24,2])
 ItkStmtListLt(L0(16_0),[25,2])
 ItkStmtListRt(L0(16_0),[25,2])
 ItkModuleRt(L1(16_1),[1,0])
 ItkRightEnd()
 ItkBOF()
     0
