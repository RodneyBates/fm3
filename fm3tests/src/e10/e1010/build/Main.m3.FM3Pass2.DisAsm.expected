   387 16_00000000000000d4                   212 ItkEOF()
   385 16_00000000000000d5                   213 ItkLeftEnd()
   383 16_00000000000000e1                   225 ItkModuleLt(L2(16_2),[1,0])
   378 16_00000000000000f0                   240 ItkDeclScopeLt(L1(16_1))
   375 16_00000000000000f2                   242 ItkOpenScopeLt(L1(16_1))
   372 16_0000000000000253                   595 ItkOpenDeclListLt(L8(16_8),[5,2])
   367 16_000000000000014c                   332 ItkTypeDeclLt([5,7])
   363 16_00000000000000f7                   247 ItkDeclNo(D5,[5,7])
   358 16_0000000000000150                   336 ItkTypeDeclType([5,9])
   354 16_00000000000000f8                   248 ItkDefTopExprNo(L17(16_11),[5,11])
   349 16_000000000000014f                   335 ItkTypeDeclRt([5,7])
   345 16_0000000000000257                   599 ItkOpenDeclListSep(L1(16_1),[5,2])
   340 16_000000000000014c                   332 ItkTypeDeclLt([7,7])
   336 16_00000000000000f7                   247 ItkDeclNo(D6,[7,7])
   331 16_0000000000000150                   336 ItkTypeDeclType([7,9])
   327 16_0000000000000180                   384 ItkSubscriptTypeAbsent([7,17])
   323 16_00000000000000f8                   248 ItkDefTopExprNo(L15(16_f),[7,11])
   318 16_000000000000014f                   335 ItkTypeDeclRt([7,7])
   314 16_0000000000000257                   599 ItkOpenDeclListSep(L2(16_2),[5,2])
   309 16_000000000000014c                   332 ItkTypeDeclLt([9,7])
   305 16_00000000000000f7                   247 ItkDeclNo(D7,[9,7])
   300 16_0000000000000150                   336 ItkTypeDeclType([9,9])
   296 16_0000000000000180                   384 ItkSubscriptTypeAbsent([9,17])
   292 16_00000000000000f8                   248 ItkDefTopExprNo(L13(16_d),[9,11])
   287 16_000000000000014f                   335 ItkTypeDeclRt([9,7])
   283 16_0000000000000257                   599 ItkOpenDeclListSep(L3(16_3),[5,2])
   278 16_000000000000014c                   332 ItkTypeDeclLt([11,7])
   274 16_00000000000000f7                   247 ItkDeclNo(D9,[11,7])
   269 16_0000000000000150                   336 ItkTypeDeclType([11,9])
   265 16_0000000000000180                   384 ItkSubscriptTypeAbsent([11,17])
   261 16_00000000000000f8                   248 ItkDefTopExprNo(L11(16_b),[11,11])
   256 16_000000000000014f                   335 ItkTypeDeclRt([11,7])
   252 16_0000000000000257                   599 ItkOpenDeclListSep(L4(16_4),[5,2])
   247 16_000000000000014c                   332 ItkTypeDeclLt([13,7])
   243 16_00000000000000f7                   247 ItkDeclNo(D11,[13,7])
   238 16_0000000000000150                   336 ItkTypeDeclType([13,11])
   234 16_00000000000000f0                   240 ItkDeclScopeLt(L2(16_2))
   231 16_00000000000001a7                   423 ItkFieldDeclListLt(L2(16_2),[14,6])
   226 16_00000000000001cf                   463 ItkFieldDeclLt([14,6])
   222 16_000000000000022b                   555 ItkFieldDeclIdListLt(L1(16_1),[14,6])
   217 16_00000000000000f7                   247 ItkDeclNo(D3,[14,6])
   212 16_000000000000022e                   558 ItkFieldDeclIdListRt(L1(16_1),[14,6])
   207 16_00000000000001d3                   467 ItkFieldDeclType([14,6])
   203 16_00000000000000f8                   248 ItkDefTopExprNo(L10(16_a),[14,11])
   198 16_00000000000001d6                   470 ItkFieldDeclValue([14,9])
   194 16_00000000000001e4                   484 ItkDeclValAbsent([15,4])
   190 16_00000000000001d2                   466 ItkFieldDeclRt([14,6])
   186 16_00000000000001ab                   427 ItkFieldDeclListSep(L1(16_1),[15,4])
   181 16_00000000000001cf                   463 ItkFieldDeclLt([15,6])
   177 16_000000000000022b                   555 ItkFieldDeclIdListLt(L1(16_1),[15,6])
   172 16_00000000000000f7                   247 ItkDeclNo(D4,[15,6])
   167 16_000000000000022e                   558 ItkFieldDeclIdListRt(L1(16_1),[15,6])
   162 16_00000000000001d3                   467 ItkFieldDeclType([15,6])
   158 16_00000000000000f8                   248 ItkDefTopExprNo(L9(16_9),[15,11])
   153 16_00000000000001d6                   470 ItkFieldDeclValue([15,9])
   149 16_00000000000001e4                   484 ItkDeclValAbsent([16,4])
   145 16_00000000000001d2                   466 ItkFieldDeclRt([15,6])
   141 16_00000000000001aa                   426 ItkFieldDeclListRt(L2(16_2),[14,6])
   136 16_00000000000000f1                   241 ItkDeclScopeRt(L2(16_2))
   133 16_00000000000000f8                   248 ItkDefTopExprNo(L8(16_8),[13,13])
   128 16_000000000000014f                   335 ItkTypeDeclRt([13,7])
   124 16_0000000000000257                   599 ItkOpenDeclListSep(L5(16_5),[5,2])
   119 16_000000000000014c                   332 ItkTypeDeclLt([18,7])
   115 16_00000000000000f7                   247 ItkDeclNo(D10,[18,7])
   110 16_0000000000000150                   336 ItkTypeDeclType([18,9])
   106 16_0000000000000180                   384 ItkSubscriptTypeAbsent([18,17])
   102 16_00000000000000f8                   248 ItkDefTopExprNo(L6(16_6),[18,11])
    97 16_000000000000014f                   335 ItkTypeDeclRt([18,7])
    93 16_0000000000000257                   599 ItkOpenDeclListSep(L6(16_6),[5,2])
    88 16_000000000000014c                   332 ItkTypeDeclLt([20,7])
    84 16_00000000000000f7                   247 ItkDeclNo(D8,[20,7])
    79 16_0000000000000150                   336 ItkTypeDeclType([20,9])
    75 16_0000000000000180                   384 ItkSubscriptTypeAbsent([20,17])
    71 16_00000000000000f8                   248 ItkDefTopExprNo(L4(16_4),[20,11])
    66 16_000000000000014f                   335 ItkTypeDeclRt([20,7])
    62 16_0000000000000257                   599 ItkOpenDeclListSep(L7(16_7),[5,2])
    57 16_000000000000014c                   332 ItkTypeDeclLt([22,7])
    53 16_00000000000000f7                   247 ItkDeclNo(D12,[22,7])
    48 16_0000000000000150                   336 ItkTypeDeclType([22,9])
    44 16_00000000000000f8                   248 ItkDefTopExprNo(L1(16_1),[22,11])
    39 16_000000000000014f                   335 ItkTypeDeclRt([22,7])
    35 16_0000000000000256                   598 ItkOpenDeclListRt(L8(16_8),[5,2])
    30 16_00000000000000e5                   229 ItkModuleBegin(L2(16_2),[24,2])
    25 16_000000000000025b                   603 ItkStmtListLt(L0(16_0),[25,2])
    20 16_000000000000025e                   606 ItkStmtListRt(L0(16_0),[25,2])
    15 16_00000000000000f3                   243 ItkOpenScopeRt(L1(16_1))
    12 16_00000000000000f1                   241 ItkDeclScopeRt(L1(16_1))
     9 16_00000000000000e4                   228 ItkModuleRt(L2(16_2),[1,0])
     4 16_00000000000000d6                   214 ItkRightEnd()
     2 16_00000000000000d3                   211 ItkBOF()
     0
